src/packages/user2/tsconfig.esm.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like you might be missing `references`?  They're not inherited via `extends` and, even if they were, you want the `esm` version, right?

Reply:
Yes, forgot. Thanks for this comment, ended up spotting some issues in my real repo due to this.